workspace/techsupport/jobs.py

```diff
@@ -0,0 +1,79 @@
+# noqa: INP001
+import json
+from datetime import datetime
+from pathlib import Path
+
+
+DATE_FMT = "%Y-%m-%d"
+
+
+def config_file():
+    return Path(__file__).parent / "ooo.json"
+
+
+def today():
+    return datetime.today().date()
+
+
+def convert_date(date_string):
+    return datetime.strptime(date_string, DATE_FMT).date()
+
+
+def get_dates_from_config():
+    start = None
+    end = None
+    config = config_file()
+    if config.exists():
+        config_dict = json.load(config_file().open())
+        start = convert_date(config_dict["start"])
+        end = convert_date(config_dict["end"])
+    return start, end
+
+
+def out_of_office_on(start_date, end_date):
+    # convert dates to ensure they're valid
+    start = convert_date(start_date)
+    end = convert_date(end_date)
+    config = {"start": start_date, "end": end_date}
+
+    # make sure the dates make sense
+    if start > end:
+        return "Error: start date must be before end date"
+    elif end < today():
+        return "Error: Can't set out of office in the past"
+
+    with open(config_file(), "w") as outfile:
+        json.dump(config, outfile)
+    if start <= today():
+        return f"Tech support out of office now ON until {end_date}"
+    return f"Tech support out of office scheduled from {start_date} until {end_date}"
+
+
+def out_of_office_off():
+    config = config_file()
+    start, _ = get_dates_from_config()
+    if config.exists():
+        config.unlink()
```

Comment:
Is it worth collapsing these lines down into `config.unlink(missing_ok=True)`?

Reply:
Now that I know that exists it is :)